hasura/metadata/databases/default/tables/public_invite.yaml

```
@@ -21,7 +21,32 @@ select_permissions:
         - email
         - expires_at
       filter: {}
+  - role: user
+    permission:
+      columns:
+        - email
+        - expires_at
+        - id
+        - team_id
+      filter:
+        team:
+          memberships:
+            _and:
+              - team_id:
+                  _eq: X-Hasura-Team-Id
+              - role:
+                  _eq: OWNER
 delete_permissions:
   - role: service
     permission:
       filter: {}
+  - role: user
+    permission:
+      filter:
+        team:
+          memberships:
+            _and:
+              - team_id:
+                  _eq: X-Hasura-Team-Id
+              - role:
+                  _eq: OWNER
```

need to add user_id to the and
```suggestion
                  _eq: OWNER
                - role:
                   _eq: x-hasura-user-id
```